code/scripting/api/objs/ship.cpp

```diff
@@ -831,7 +832,7 @@ ADE_VIRTVAR(Team, l_Ship, "team", "Ship's team", "team", "Ship team, or invalid
 	return ade_set_args(L, "o", l_Team.Set(shipp->team));
 }
 
-ADE_VIRTVAR(PersonaIndex, l_Ship, "number", "Persona index", "number", "The index of the persona from messages.tbl, 0 if no persona is set")
+ADE_VIRTVAR_DEPRECATED(PersonaIndex, l_Ship, "number", "Persona index", "number", "The index of the persona from messages.tbl, 0 if no persona is set", gameversion::version(25, 0), "Deprecated in favor of Persona")
```

Comment:
Ah sorry, looking at existing script plans with index tracking, it might be useful to not deprecate this since the `l_Persona` script object doesn't have a way to access the underlying index (similar ways in which `getShipClassIndex` has proved useful). If other folks would prefer to deprecate this though then I certainly understand!